src/main/java/nl/ordina/jobcrawler/service/ScraperService.java

```diff
@@ -46,7 +46,7 @@ public ScraperService(VacancyService vacancyService, MatchSkillsService matchSki
         }
     };
 
-    @PostConstruct
+    //    @PostConstruct
```

Comment:
do we need to comment this?

Reply:
We needed to scrape on startup when testing some saving and scraping functionality, however that is not needed anymore and it hinders the debugging. So for now it can be commented until we implement the functionality to scrape when the data is older than 6 hours, which is already in the backlog.